microservice_templates/microservice_node_template/supervisor/_MICROSERVICE_NODE_TEMPLATE_.conf

```diff
@@ -0,0 +1,3 @@
+[program:_MICROSERVICE_NODE_TEMPLATE_]
+directory=/opt/_MICROSERVICE_NODE_TEMPLATE_/src
+command=nodejs _MICROSERVICE_NODE_TEMPLATE_.js
```

Comment:
you can change command to npm start, as you have start script in package.json file